dev/tasks/r/github.packages.yml

```diff
@@ -56,6 +56,57 @@ jobs:
           name: r-pkg__src__contrib
           path: arrow/r/arrow_*.tar.gz
 
+  macos-cpp:
+    name: C++ Binary macOS OpenSSL {{ '${{ matrix.openssl }}' }} {{ '${{ matrix.platform.arch }}'}}
+    runs-on: {{ '${{ matrix.platform.runs_on }}'}}
+    needs: source
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - { runs_on: ["self-hosted", "macos-10.13"],  arch: "x86_64" }

```

Comment:
Does this 10.13 build for x86 cover all x86 needs (e.g. for macOS 12, 13, 14)?

Reply:
Yes, the binaries are upwards compatible that's why we always build on the lowest possible version.